### PR TITLE
Add intent configuration modal on konnector install

### PR DESCRIPTION
**Pull request by CPatchane**
Waiting for the next release of cozy-ui to handle `<IntentModal />`

### CODE REVIEW

**Comment by CPatchane** [comment]
[(2) ETU j'ai la modale d'inter-app de collect à la fin de l'installation d'un connecteur](https://trello.com/c/470Nuja1/895-2-etu-jai-la-modale-dinter-app-de-collect-%C3%A0-la-fin-de-linstallation-dun-connecteur)

File: src/ducks/apps/components/ApplicationRouting.jsx [hunk 1]
```diff
@@ -60,13 +64,37 @@ export class ApplicationRouting extends Component {
...
-                    isFetching={isFetching}
                   />
                 )
               }
             }
           }}
         />
+        <Route
+          path={`/${parent}/:appSlug/configure`}
+          render={({ match }) => {
```
Comment (gregorylegarec) [inline src/ducks/apps/components/ApplicationRouting.jsx]: To make the code a little bit clearer, do you think it's interesting to externalize the `render` method here ? It fees like a pretty huge block of code to be just in a Component's property.

File: src/ducks/apps/components/InstallModal.jsx [hunk 2]
```diff
@@ -19,11 +19,15 @@ export class InstallModal extends Component {
...
   installApp () {
     this.setState({ error: null })
-    const { app } = this.props
+    const { app, parent, history } = this.props
     this.props
       .installApp(app.slug, app.type)
       .then(() => {
-        this.gotoParent()
+        if(app.type === APP_TYPE.KONNECTOR) {
+          history.push(`${parent}/${app.slug}/configure`)
```
Comment (gregorylegarec) [inline src/ducks/apps/components/InstallModal.jsx]: 👍

File: src/ducks/apps/components/ApplicationRouting.jsx (hunk 1 above)
Comment (CPatchane) [reply src/ducks/apps/components/ApplicationRouting.jsx]: I don't know, these parts seems similar but not enough to be externalized I think

**Comment by gregorylegarec** [comment]
Thanks @CPatchane 👌

**Pull request merged**